backend/src/main/java/co/inventorsoft/scripty/model/entity/User.java

```diff
@@ -36,8 +37,9 @@
     @Column(length = 60)
     String password;
 
-    @Lob
-    Byte picture;
+    @Column(columnDefinition = "TEXT")
+    @Convert(converter = PictureConverter.class)
+    PictureDto picture;
```

Comment:
Why don't simply have it as two columns in database? If you want to work with `PictureDto` you can use `@Embedded` from Hibernate. [Example](https://www.concretepage.com/hibernate/example-embeddable-embedded-hibernate-annotation)